posthog/async_migrations/runner.py

```diff
@@ -173,15 +170,9 @@ def attempt_migration_rollback(migration_instance: AsyncMigration, force: bool =
     try:
         ops = get_async_migration_definition(migration_instance.name).operations
 
-        i = len(ops) - 1
-        for op in reversed(ops[: migration_instance.current_operation_index + 1]):
-            if not op.rollback:
-                if op.rollback == "":
-                    continue
-                raise Exception(f"No rollback provided for operation at index {i}: {op.sql}")
```

Comment:
Note that sometimes the rollback is do nothing & that's the new default. All operations should have a rollback, if they don't however given that we can now provide random functions the `rollback_fn` could raise an Exception.

Reply:
That's why this error was here. To force people to write a rollback or rather override it explicitly to a no-op